index.html

```diff
@@ -29,15 +29,182 @@ <h1>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </div>
+                    <button>See project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-image">
+                 <img src="./images/card3img.png" alt="">   
+                </div>
+                <div class="card-text">
+                    <h3>Tonic</h3>
+                    <div class="expertise">
+                        <ul>
+                            <li>CANOPY</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="programes">
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </div>
+                    <button>See project</button>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-image">
+                 <img src="./images/card4img.png" alt="">   
+                </div>
+                <div class="card-text">
+                    <h3>Multi-Post <span>Stories</span></h3>
+                    <div class="expertise">
+                        <ul>
+                            <li>CANOPY</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li class="designed-color">Back End Dev</li>
+                            <li><img src="./images/Counter.png" alt=""></li>
+                            <li class="designed-color">2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="programes">
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </div>
+                    <button>See project</button>
+                </div>
+            </div>
+        </section> 
+        <section class="about-me">
+            <h2>About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
```

Comment:
- [ ] [OPTIONAL] I am requesting this change as **optional** since the previous reviewer did not mention it 👁️ 👁️  **BUT** I highly advise implementing it. Because it would only enhance the quality of your work 😉 
In this section, your design does not match the design from Figma. So kindly fix that 🙏🏽 

Your design 👇🏽 

![Screenshot from 2023-06-03 23-23-05](https://github.com/AbuTalha3/Portfolio-Setup/assets/10439283/6c004a90-2c16-40df-a188-7e757a091836)

Design from Figma 👇🏽 

![Screenshot from 2023-06-03 23-29-51](https://github.com/AbuTalha3/Portfolio-Setup/assets/10439283/4cccb16d-c4a5-4ef5-9296-dfaa2f429ef5)